app/modules/clustersync/SectionSyncUpdateProcessor.scala

```diff
@@ -7,25 +7,40 @@ import play.api.Logger
 import repositories.SectionLookupCache
 import services.KinesisStreamRecordProcessor
 
-object SectionSyncUpdateProcessor extends KinesisStreamRecordProcessor {
+import scala.util.{Failure, Success, Try}
+
+object SectionEventDeserialiser {
 
   import services.KinesisRecordPayloadConversions._
 
-  override def process(record: Record) {
-    val sectionEvent = SectionEvent.decode(record)
+  def deserialise(record: Record): Try[SectionEvent] = {
+    val tProto = kinesisRecordAsThriftCompactProtocol(record, stripCompressionByte = true)
```

Comment:
Can we add a comment around why we're setting `stripCompressionByte` to true here?

Reply:
There is a test that checks this behaviour which kinda acts as a comment, but can't hurt.

Reply:
yeah i can add that, test handles that but not say about context of this work-around as it is in some way a work-around

Reply:
added